CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
+4.74.0
+======
+
+### Features
+* Deprecate ROPC flow in Public Client Applications https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5355.
+* AuthenticationResult exposes a new BindingCertificate property that returns the X.509 certificate bound to the access token in mTLS-PoP scenarios. https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5370.
+
+### Bug fixes
+* MSAL now honors the DEFAULT_IDENTITY_CLIENT_ID environment variable when acquiring tokens from Azure Machine Learning managed-identity endpoint. https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5350.
```

Comment:
Did you mean to link to the PR? https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5351

Or is linking to the issue fine?

Reply:
We always link to the issue. But for the Public Client App we did not have an issue so linked to the PR